lib/foreman/renderer.rb

```diff
@@ -232,6 +275,11 @@ def load_template_vars
       @template_url = params['url']
     end
 
+    def global_setting(name, default = nil)
+      raise(FilteredGlobalSettingAccessed, _('Global setting "%s" is not accessible in safe-mode') % name) if Setting[:safemode_render] && !ALLOWED_GLOBAL_SETTINGS.include?(name.to_sym)
+      Setting[name.to_sym].blank? ? default : Setting[name.to_sym]
```

Comment:
this does not work well with false (valid value) which is considered blank, I guess the behavior should differ if settings_type is boolean